### ISSUE TITLE
Linter JS plugins: Include JS rules in rule count in "Finished" message

### ISSUE THREAD

**Issue by overlookmotel** [oxc-project/oxc#12433]
Config:

```js
{
  "plugins": [ "../plugins/donkey.js" ],
  "categories": { "correctness": "off" },
  "rules": {
    "donkey/no-donkeys": "error",
  }
}
```

Output:

```
Finished in 559ms on 6057 files with 0 rules using 14 threads.
//                                   ^^^^^^^
```

External rules should be included in the rules count.

**Comment by camchenry** [comment]
Alternatively: should we remove this rule count? Whenever we add a new default rule it creates a large diff in the snapshots. It's a less useful number now as it is because of overrides and nested configs changing the rule count on a per file basis. On the other hand, it does contextualize the performance, so maybe it is worth keeping.

Edit: maybe we could display throughout numbers instead as an alternative? Like "finished X files in Y ms (NNN MB/s).

**Comment by Sysix** [comment]
+1 for removing this info.

The number will never match the real rules count
Rules can be enabled/disabled via
- Nested Folders 
- `override` Configuration key
- `extends` Configuration Key

Some rules will be later filtered with `should_run`.

**Comment by camchenry** [comment]
> Alternatively: should we remove this rule count?

Actually, I have a better suggestion: we should show even more data here to make it clear this is normally a range of values (not a singular rule count) More info in https://github.com/oxc-project/oxc/issues/12444.
